### PR TITLE
Fix formatting nits in arm tutorial

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

**Review by npentrel — APPROVED** [review]

File: docs/tutorials/motion/accessing-and-moving-robot-arm.md [hunk 1]
```diff
@@ -44,23 +44,23 @@ If you are connecting to a real robotic arm during this tutorial, make sure your
...
 5. Under the **COMPONENTS** section, create a component with the following attributes:
 
-* Choose `Arm` as the **Type** selection
-* Choose your desired model in the **Model** selection
-  * If you're using an xArm 6, choose the `xArm6` model from the drop-down list
-* Enter `myArm` as the **Name** for this component
+  * Choose `Arm` as the **Type** selection
+  * Choose your desired model in the **Model** selection
+    * If you're using an xArm 6, choose the `xArm6` model from the drop-down list
+  * Enter `myArm` as the **Name** for this component
```
Comment (JessamyT) [inline docs/tutorials/motion/accessing-and-moving-robot-arm.md:50]:
```suggestion
    * Choose `Arm` as the **Type** selection
    * Choose your desired model in the **Model** selection
      * If you're using an xArm 6, choose the `xArm6` model from the drop-down list
    * Enter `myArm` as the **Name** for this component
```

File: docs/tutorials/motion/accessing-and-moving-robot-arm.md [hunk 2]
```diff
@@ -44,23 +44,23 @@ If you are connecting to a real robotic arm during this tutorial, make sure your
...
 
 5. In the newly created `myArm` component panel, fill in some additional details:
 
-* Enter the IP address for your robot controller in the **Host** field
-* Enter a desired velocity *(in degrees per second)* for the robot arm's motions in the **Speed** field
-  * For an `xArm6`, a safe value is `15` degrees per second.
+  * Enter the IP address for your robot controller in the **Host** field
+  * Enter a desired velocity *(in degrees per second)* for the robot arm's motions in the **Speed** field
+    * For an `xArm6`, a safe value is `15` degrees per second.
```
Comment (JessamyT) [inline docs/tutorials/motion/accessing-and-moving-robot-arm.md:56]:
```suggestion
    * Enter the IP address for your robot controller in the **Host** field
    * Enter a desired velocity *(in degrees per second)* for the robot arm's motions in the **Speed** field
      * For an `xArm6`, a safe value is `15` degrees per second.
```

File: docs/tutorials/motion/accessing-and-moving-robot-arm.md (hunk 1 above)
Comment (JessamyT) [reply docs/tutorials/motion/accessing-and-moving-robot-arm.md]: Had to add more spaces for linter

**Comment by JessamyT** [comment]
In my local build it now formats correctly but not in the rendering

**Pull request merged**